packages/angular_devkit/benchmark/src/main_spec.ts

```diff
@@ -142,4 +145,51 @@ describe('benchmark binary', () => {
     stdout.lines.forEach(line => expect(line).toMatch(/^\[abc\]/));
     expect(res).toEqual(0);
   });
+
+  it('uses watch-script and watch-matcher', async () => {
+    const args = [
+      '--watch-matcher',
+      'Complete',
+      '--watch-script',
+      watchTriggerScript,
+      '--',
+      'node',
+      benchmarkWatchScript,
+    ];
+    const res = await main({ args, stdout, stderr });
+    expect(stdout.lines).toContain('[benchmark] Process Stats\n');
+    expect(res).toEqual(0);
+  }, 30000);
+
+  it('should not fail with exit code', async () => {
```

Comment:
Maybe `should not fail with exit code` -> `should fail with exit code 1 when retries are exceeded`?

Reply:
Actually, the benchmarker will not return the exit code in the error message. That was actually